composer.json

```diff
@@ -25,8 +25,7 @@
         "php-coveralls/php-coveralls": "^2.1",
         "php-http/guzzle6-adapter": "^2.0",
         "phpunit/phpunit": "^8.0",
-        "squizlabs/php_codesniffer": "^3.4",
-        "symfony/phpunit-bridge": "^5.0"
```

Comment:
All tests fail now with

All tests fail now with
```
Class "Symfony\Bridge\PhpUnit\SymfonyTestsListener" does not exist
```

Reply:
forgot travis has it's own phpunit confix.
fixed